### ISSUE TITLE
2020-04-06 データ更新

### ISSUE THREAD

**Issue by jacoyutorius** [code-for-hamamatsu/covid19#97]
## 改善詳細 / Details of Improvement
- 2020/04/06 ぶんのデータを反映させる

**Comment by euledge** [comment]
このデータって手で作ってるのかな？もしrubyとかのスクリプトで作れるなら、news.jsonでやったみたいに更新できると思いますよ

**Comment by jacoyutorius** [comment]
ありがとうございます〜、data.jsonは別の方が担当してくれてます◎ そろそろ組み込めそうな感じ。
そのへんのやりとりはJAWS-UG浜松のSlackチャンネルでやってますのでよろしければ見てみてくださいmm

https://join.slack.com/t/jaws-ug-hamamatsu/shared_invite/zt-dc5cgs87-cNw8QOxXeqhJnB8YB8A6Uw
